src/WordPress/Hooks.php

```diff
@@ -259,7 +259,9 @@ public function getPostRelatedLinks($postId)
             $attachmentUrls = array();
             foreach (get_intermediate_image_sizes() as $size) {
                 $attachmentSrc = wp_get_attachment_image_src($postId, $size);
-                $attachmentUrls[] = $attachmentSrc[0];
+                if ($attachmentSrc) {
```

Comment:
let's be a little more explicit here as the return type is either array or boolean. it's not immediately clear from the code that `$attachmentSrc` can be either and it may throw some people off. 

```suggestion
                if (is_array($attachmentSrc) && !empty($attachmentSrc)) {
```